stacker/hooks/aws_lambda.py

```diff
@@ -93,6 +93,18 @@ def _calculate_hash(files, root):
     return file_hash.hexdigest()
 
 
+def _calculate_prebuilt_hash(f):
```

Comment:
I think we can just re-use [_calculate_hash](https://github.com/remind101/stacker/blob/e123052c6f549559bea1426983786803011f72eb/stacker/hooks/aws_lambda.py#L73-L93).

```python
_calculate_hash("path/to/zip-file.zip")
```

Reply:
That won't be the exact MD5 of the file, as it will append a null character to the stream when calculating.

Reply:
Is that a problem? We just need a consistent hash.

Reply:
Probably not, but it seemed more natural to me (as I just used `md5sum` to figure out the reference values).